tests/test_matrix_utils.py

```
@@ -0,0 +1,317 @@
+
+
+@patch("biblebot.auth.os.path.exists")
+@patch("biblebot.auth.load_credentials")
+def test_load_credentials_file_not_exists(mock_load, mock_exists):
+    """Test credentials loading when file doesn't exist."""
+    mock_exists.return_value = False
+    mock_load.return_value = None  # Override the fixture
+
+    credentials = mock_load()
+
+    assert credentials is None
```

Ruff (0.12.2)

Use of `assert` detected

(S101)